src/main/java/devcource/hihi/boardjpa/controller/PostRestController.java

```diff
@@ -0,0 +1,51 @@
+package devcource.hihi.boardjpa.controller;
+
+import devcource.hihi.boardjpa.domain.Post;
+import devcource.hihi.boardjpa.dto.post.CreatePostDto;
+import devcource.hihi.boardjpa.dto.post.PageDto;
+import devcource.hihi.boardjpa.dto.post.UpdatePostDto;
+import devcource.hihi.boardjpa.dto.post.ResponsePostDto;
+import devcource.hihi.boardjpa.service.PostService;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/posts")
+public class PostRestController {
+
+    private final PostService postService;
+
+    public PostRestController(PostService postService) {
+        this.postService = postService;
+    }
+    @GetMapping
```

Comment:
```suggestion
    }
    
    @GetMapping
```

p1; 
띄워주면 좀 더 깔끔해보일 듯 합니다.